commit cfa29b46af94d83e31aa000dbaee553f9c4d670c
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Fri Jul 14 12:59:55 2017 +0100

    bcm2835-mmc: Fix DMA usage
    
    The previous change ("bcm2835-mmc: Only claim one DMA channel")
    used an incorrect variable, the effect of which was to prevent
    DMA from being used at all. Fix that bug by using the right
    variable.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/drivers/mmc/host/bcm2835-mmc.c b/drivers/mmc/host/bcm2835-mmc.c
index 4fe8d1fe4457..981db05de1ff 100644
--- a/drivers/mmc/host/bcm2835-mmc.c
+++ b/drivers/mmc/host/bcm2835-mmc.c
@@ -1354,14 +1354,14 @@ static int bcm2835_mmc_add_host(struct bcm2835_host *host)
 		if (ret == 0) {
 			host->dma_cfg_rx = cfg;
 
-			host->use_dma = true;
+			host->have_dma = true;
 		} else {
 			pr_err("%s: unable to configure DMA channel. "
-			       "Faling back to PIO\n",
+			       "Falling back to PIO\n",
 			       mmc_hostname(mmc));
 			dma_release_channel(host->dma_chan_rxtx);
 			host->dma_chan_rxtx = NULL;
-			host->use_dma = false;
+			host->have_dma = false;
 		}
 	}
 #endif
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1585_cfa29b46af94d83e31aa000dbaee553f9c4d670c.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1585_cfa29b46af94d83e31aa000dbaee553f9c4d670c.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
